static-site/src/layouts/userDataWrapper.jsx

```diff
@@ -11,7 +11,7 @@ export default class UserDataWrapper extends React.Component {
   }
 
   loadUser() {
-    console.log("Fetching user data");
+    // console.log("Fetching user data");
```

Comment:
Looks like an unrelated change made during dev slipped into this commit.

Reply:
resolved